packages/nvidia_nat_langchain/src/nat/plugins/langchain/langgraph_workflow.py

```
@@ -0,0 +1,162 @@
+        # Convert the value to message format using LangChain utils. Ensures is compatible with the message format
+        messages = convert_to_messages(value)
+
+        return LanggraphWrapperInput(messages=messages)
+
+    async def _ainvoke(self, value: LanggraphWrapperInput) -> LanggraphWrapperOutput:
+
+        try:
+            output = await self._graph.ainvoke(value.model_dump())
+            return LanggraphWrapperOutput.model_validate(output)
+        except Exception as e:
+            raise RuntimeError(f"Error in langchain graph: {e}") from e
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)